src/Sentry.php

```diff
@@ -50,6 +50,7 @@ public static function initializeOnce(): void
         unset($options['typo3_integrations']);
         $httpOptions = $GLOBALS['TYPO3_CONF_VARS']['HTTP'];
         $httpOptions['verify'] = filter_var($httpOptions['verify'], FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE) ?? $httpOptions['verify'];
+        if (empty($httpOptions['handler'])) unset($httpOptions['handler']);
```

Comment:
if without braces is not conform with PSR-2

Reply:
Agree. Besided that, why is this unset needed anyway and how is it related to TYPO3 10.4?